service/controller/key/key.go

```diff
@@ -21,10 +24,36 @@ const (
 	ServiceTypeManaged = "managed"
 )
 
-func ReleaseVersion(customResource releasev1alpha1.Release) string {
-	return customResource.Spec.Version
+// ReleaseAppCRName returns the name of the release App CR for the given release cycle.
+func ReleaseAppCRName(releaseCycleCR releasev1alpha1.ReleaseCycle) string {
+	return ReleasePrefix(releaseCycleCR.GetName())
+}
+
+// ReleasePrefix adds release- prefix to name.
+func ReleasePrefix(name string) string {
+	return fmt.Sprintf("release-%s", name)
+}
```

Comment:
Same for this. I don't think it's useful to expose that in the key package. We should have stuff shared between resources in here.

Reply:
move this function to `app` resource.